leveldb-firebase.podspec

```diff
@@ -0,0 +1,58 @@
+# CocoaPods 1.x currently does not support libraries with C++ headers. See
+# https://github.com/CocoaPods/CocoaPods/issues/5152. The workaround is to use
+# the 1.2.1.beta.1 with its new option --skip-import-validation.
+
+# To update the pod:
```

Comment:
This is fantastic commentary, but given its volume I wonder if at this point this wouldn't be better in the `README.md`?